tests/smoke/Makefile

```diff
@@ -1,4 +1,12 @@
-test:
+SHELL:=/bin/bash
+
+deps:
+	git clone https://github.com/sstephenson/bats.git
+	mkdir ~/bin || :
```

Comment:
On Travis CI, we have no promises that the `travis` user has permissions to write to `/usr/local/...`. Even when installing via `sudo`  I ran into this issue, and related permissions problems around it. In the end I opted to use a custom bin directory, which is portable to all linux distros anyway.